README.md

```diff
@@ -125,16 +122,21 @@ entity.native$( function (err, client, releaseConnection){
 } )
 ```
 
-## Running tests
+## Contributing
+The [Senecajs org][] encourages open participation. If you feel you
```

Comment:
should be like in the header `Seneca.js`: [Seneca.js](http://senecajs.org) not `Senecajs org`